React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -412,6 +418,17 @@ - (void)finalizeUpdates:(RNComponentViewUpdateMask)updateMask
   [self invalidateLayer];
 }
 
+- (void)announceForAccessibilityWithOptions:(NSString*)announcement
+{
+  if (@available(iOS 11.0, *)) {
+    BOOL accessibilityAnnouncementNotEmpty = [announcement length] != 0;
+    if (accessibilityAnnouncementNotEmpty) {
+      UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, announcement);
+      _triggerAccessibilityAnnouncement = NO;
+    }
+  }
+}
+
```

Comment:
This method is shared with PR https://github.com/facebook/react-native/pull/35432/files#diff-ce3cb56ee74b04a10d3bec55e1d0a0eda2f3cfbd23f6024a3669fbe3d2e83337R458-R472